addons/sourcemod/scripting/scp_sf/scps/173.sp

```diff
@@ -55,6 +55,9 @@ public bool SCP173_Create(int client)
 		SetEntProp(weapon, Prop_Send, "m_iAccountID", GetSteamAccountID(client, false));
 		SetEntPropEnt(client, Prop_Send, "m_hActiveWeapon", weapon);
 	}
+	
+	// no crouching
+	TF2Attrib_SetByDefIndex(client, 820, 1.0);	
```

Comment:
This doesn't get reset on changing classes or round restart, probably better too have it on the weapon. There's also a way to fix crouching via `SetEntProp(client, Prop_Send, "m_bDucked", true);` before teleporting which I forgot to add.

Reply:
Fixed, tested and works fine on the weapon. I disabled crouching entirely for peanut as a tall statue shouldn't be able to fit inside a tiny vent